core/interop/source-generation/custom-marshalling/src/nativelib/nativelib.csproj

```diff
@@ -0,0 +1,94 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Thoughts on using DNNE for this instead? Seems like the native part actually being C+++ isn't all that interesting.

Reply:
For the purposes of a sample demonstrating normal usage, I think it makes more sense to have it in C++ instead of introducing DNNE/hosting going back to C#.